### ISSUE TITLE
No message in body for non-existent endpoint

### ISSUE THREAD

**Issue by aliaksiejmaroz** [simpleidserver/SimpleIdServer#417]
**Steps to reproduce:**
1. Send request to endpoint not listed in `SCIMEndpoints`.
2. Get response with 404 status code without body.

**Expected result:**
Standard body with json should be returned in case of 4xx errors (according to [RFC-7644, 3.12 HTTP Status and Error Response Handling](https://www.rfc-editor.org/rfc/rfc7644#section-3.12))
```
{
    "schemas": [
        "urn:ietf:params:scim:api:messages:2.0:Error"
    ],
    "status": "404",
    "scimType": "unknown",  
    "detail": "some text"
}
```

**Comment by simpleidserver** [comment]
The issue is fixed in the branch `release/v2.0.26`.

KR,

SID

**Comment by aliaksiejmaroz** [comment]
Thank you for fixes, I have few issue now:
1. Now I can reproduce [this one already fixed issue](https://github.com/simpleidserver/SimpleIdServer/issues/390) for these changes.
2. **Post/Patch/Delete** requests to _unknown_ endpoint return **405** status code with an empty body. As I know it should be **404** with not empty body and with '_scimType_' in it.

**Comment by simpleidserver** [comment]
Indeed the `DefaultController` was not complete.
The controller has been updated in the branch `release\v3.0.2`, and the issue is fixed

**Comment by aliaksiejmaroz** [comment]
There is no **PATCH** in `DefaultController`. Also there are no _ScimType_ in responses for _unknown_ endpoint's responses but it should be there.

**Comment by simpleidserver** [comment]
My bad... the issue is fixed in the branch "release/v3.0.3".
The following changes have been made:
* Set "ScimType" to unknown
* Add the missing PATCH operation
